### PR TITLE
Add messmenu.html

### PR DESCRIPTION
**Pull request by PrashikD**
Page appearance corrected. Backend not yet integrated into the menu table. Page script is internally written.

### CODE REVIEW

**Comment by PrashikD** [comment]
https://1drv.ms/u/s!AkGLAIp3k2l1jg0g6MMOitsc75q6

File: newmessmenu.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+            <a href="#" data-target="mobile-demo" class="sidenav-trigger"><i class="material-icons">menu</i></a>
+            <a href="#" class="brand-logo left valign-wrapper">
+                <img src="./static/images/IIT_Kharagpur_Logo.svg" alt="" class="responsive-img">
+
+            </a>
+            <ul id="nav-mobile" class="hide-on-med-and-down right">
+                <li><a href="index.html" class="waves-effect waves-light">Home</a></li>
+                <li><a href="interiit.php" class="waves-effect">Inter IIT</a></li>
+                <li ><a href="notices.php" class="waves-effect">Notices</a></li>
+                <li class="active"><a href='events.html' class="waves-effect">Events</a></li>
```
Comment (thealphadollar) [inline newmessmenu.html]: Please create an entry for mess menu.

File: newmessmenu.html [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+                <li><a href='openiit.html' class="waves-effect">Open IIT</a></li>
+                <li><a href="gc-results.html" class="waves-effect">GC 19</a></li>
+                <li><a href="gc.php" class="waves-effect">GC</a></li>
+                <li><a href="fests.html" class="waves-effect">Fests</a></li>
+                <li><a href="societies.html" class="waves-effect">Societies</a></li>
+                <li><a href="contacts.html" class="waves-effect">Contacts</a></li>
+                <li><a href="blogpost.html" class="waves-effect">Blog</a></li>
```
Comment (thealphadollar) [inline newmessmenu.html]: This had drop down IIRC.

File: newmessmenu.html [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    </tr>
+    <tr>
+        <td>Dinner</td>
+        <td>Methi Paratha + Sabji</td>
+        <td>Masala Dosa</td>
+        <td>Sambhar</td>
+        <td>Idli </td>
+        <td>Upma</td>
+        <td>Puri + Sabji</td>
+        <td>Poha</td>
```
Comment (thealphadollar) [inline newmessmenu.html]: Fill this after getting from backend; why have you hardcoded?

Comment (thealphadollar) [reply newmessmenu.html]: For testing purposes fill the mess menu of your own hall and test it.

**Comment by PrashikD** [comment]
Delete the messmenu.js, I think it is of no use now

**Pull request merged**